### PR TITLE
Updated schema

### PR DESCRIPTION
**Pull request by wojta**
Memeolist schema is obsolete in iOS app, updated from Android app.

- [x] fixed script for code-generation 
- [x] updated schema from Android memeolist app
- [x] make it working

### CODE REVIEW

**Comment by wtrocki** [comment]
@wojta Is schema commit from android server or it's from UI ?

**Comment by wtrocki** [comment]
@wojta From what I see there is no:

- Change in actual graphql queries: 
see server side example UI that has all of those
- Change in schema (we added tons of fields)


For schema and queries see Android https://github.com/aerogear/aerogear-android-example-apps/tree/master/memeolist/app/src/main/graphql/org/aerogear/android/app/memeolist/graphqlpp which has latest version:

**Comment by wtrocki** [comment]
@wojta Let me know when this will be finished.

File: Memeolist/Memeolist/GeneratedAPI.swift [hunk 1]
```diff
@@ -2,62 +2,18 @@
 
 import Apollo
```
Comment (wtrocki) [inline Memeolist/Memeolist/GeneratedAPI.swift]: Let's add this to gitignore. This is generated and it should not be pushed back to the git.

**Comment by wtrocki** [comment]
@wojta let's remove outdated script to get recent version of the schema.

File: Memeolist/Memeolist/view/create-meme/CreateMemeController.graphql [hunk 2]
```diff
@@ -1,7 +1,9 @@
 ## Creating meme
-mutation newMeme($url: String!) {
-  createMeme(photoUrl: $url) {
-    id
-    photoUrl
-  }
+mutation CreateMeme($ownerid: ID!, $photourl: String!, $owner: String!) {
```
Comment (wtrocki) [inline Memeolist/Memeolist/view/create-meme/CreateMemeController.graphql]: Formatting?

**Comment by wtrocki** [comment]
@wojta PR is missing files. 
Please push remaining files as this is blocking other work on IOS SDK.

File: Memeolist/Memeolist/view/create-meme/CreateMemeController.swift [hunk 3]
```diff
@@ -51,8 +52,9 @@ class CreateMemeController: UIViewController,
         let url = MemeService.instance.createMemeUrl(imageUrl: rawMemeUrl,
                                                      top: self.topTextEdit.text ?? "",
                                                      bottom: self.bottomTextEdit.text ?? "")
-
-        AgsSync.instance.client?.perform(mutation: NewMemeMutation(url: url)) { result, error in
+        
+        
+        AgsSync.instance.client?.perform(mutation: CreateMemeMutation(ownerid:"1", photourl: url,owner:"???")) { result, error in
```
Comment (wtrocki) [inline Memeolist/Memeolist/view/create-meme/CreateMemeController.swift]: Put some sample owner

File: Memeolist/Memeolist/mobile-services.json [hunk 4]
```diff
@@ -4,25 +4,23 @@
     "namespace": "myproject",
```
Comment (wtrocki) [inline Memeolist/Memeolist/mobile-services.json]: Wrong formatting im this file.

**Comment by wojta** [comment]
I have not added any new files, so if it's is missing something let me know.

**Review by wtrocki — APPROVED** [review]

**Pull request merged**